src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -368,7 +368,7 @@ public World getWorld()
     }
 
     @Override
-    public float getDistanceSquared(@NotNull BlockPos pos)
+    public long getDistanceSquared(@NotNull BlockPos pos)
     {
         //  Perform a 2D distance calculation, so pass center.posY as the Y
         return BlockPosUtil.getDistanceSquared(center, new BlockPos(pos.getX(), center.getY(), pos.getZ()));
```

Comment:
This distance isn't 2D like the comment says, we should be doing a 2D calculation though. Also why are we making a new BlockPos, from an existing BlockPos? They are immutable, we should just pass in pos.